### ISSUE TITLE
Not synchronising from Endomondo to Strava and Runkeeper.

### ISSUE THREAD

**Issue by Gozzeh** [cpfair/tapiriik#238]
For a couple of days now I've noticed that activities aren't getting synchronised.  When I click the synchronise now it queues and then I check back an hour or 2 later and nothing has happened.  I currently have activities stretching back nearly 24 hours that still have not been synchronised.  

When I click all activities it shows that I have an activity from yesterday evening (nothing from this morning's) but it shows as having synchronised on Runkeeper and Strava but not Endomondo.  However the activity was recorded with Endomondo and it is not showing in Strava or Runkeeper.

**Comment by Gozzeh** [comment]
I disconnected my endomondo account and re connected it and it transferred the past few days of activities.  However this morning's activity still has not synchronised after 1.5 hours.

**Comment by jj09** [comment]
I've seen this started happening this month. My July activities (for now: 3 bike rides), are not being synced with Strava.

**Comment by Gozzeh** [comment]
So again I removed Endomondo and re-added it and it sync'd my days activities.  Nothing since though.

**Comment by samjudson** [comment]
I'm also seeing this issue between Endomondo to both Strava and Smashrun.

**Comment by tzetze** [comment]
Almost the same story for me: Endomondo -> Strava + Dropbox 
The last successful sync was at 28 Jun.

**Comment by Alderaic** [comment]
same issue endo -> connect seems to work with a lot of pushing, synced 3 times a run this morning and it does not show, got yesterday's run after a lot of manual sync (and I am an auto sync subscriber)

**Comment by samjudson** [comment]
Yeah, forgot to mention I'm a auto sync subscriber as well. I've tried manually queuing as well but that hasn't done anything.

**Comment by Gozzeh** [comment]
Same with me

**Comment by kasi1999999** [comment]
Same problem.

**Comment by froden** [comment]
Same problem here. I have to reauthorize Endomondo before every sync or nothing happens.

**Comment by gertjanstoker** [comment]
I also have this problem, I am glad to read that I am not the only one with this problem. I saw this issue occur starting July 1. I noticed I was not ranking high in the strava leaderboard. I am also a paid subscriber for automatically sync. Now I manually perform a couple of tasks to get the sync done. Very annoying. Can the owner conform that this in an issue?

**Comment by Gozzeh** [comment]
So I thought this was fixed after the weekend but it has started happening to me again :(

**Comment by tswestendorp** [comment]
Same here, my ride of today doesn't sync from Endomondo to Strava and Dropbox

**Comment by gertjanstoker** [comment]
Hmm, I had the idea that the synchronisation issue was solved. My rides of today were well synced.
